node/silkworm/db/tables.hpp

```diff
@@ -139,8 +137,8 @@ inline constexpr db::MapConfig kTables[]{
     kTxLookup,
 };
 
-// Create all tables that do not yet exist.
-void create_all(mdbx::txn& txn);
+//! \brief Ensures all defined tables are present in db with consistent flags. Should a table not exist it gets created
+void deploy_chaindata_tables(mdbx::txn& txn);
```

Comment:
Hmm. Perhaps "create_chaindata_tables"? "deploy" is less clear IMHO.

Reply:
Well actually the method does both a check (if table exist it must have compatible flags) or a creation (if table does not exist). I cant find a single word to describe it. Maybe `check_or_create_chaindata_tables` ?

Reply:
Well, IMHO simply "create" is fine and the checks are just details. For example, we typically call methods just something like "cleanup", even though `cleanup` is probably idempotent and has some checks internally against double cleanup. But I am not insisting.